app/views/exam_templates/_split_pdf_log.html.erb

```diff
@@ -1,27 +0,0 @@
-<tr>
```

Comment:
Removed this file; no longer used. (Flagged here because it was including `icons/error.png`)